src/pages/style.css

```diff
@@ -0,0 +1,19 @@
+.scrollbox{
+    overflow: hidden;
+}
+
+.scrollbox:hover{
+    overflow-y: auto;
+    overscroll-behavior: contain;
+    -ms-scroll-chaining: contain;
+}
+.scrollbox::-webkit-scrollbar {
+        width: 13px;
+        height: 15px;
+    }
+.scrollbox::-webkit-scrollbar-thumb  
+    {  
+    height: 15px;
+    border-radius: 5px;  
+    background-color: #ABABAD;  
+    }  
```

Comment:
I don't think using a global CSS is a good way to approach this request. Maybe try to define styles inside of variants in the Chakra styles folder as others do?

Reply:
It seems that  Chakra  doesn't support some props like  the -webkit-scrollbar-thumb, might find a way to update it later.